guides/common/modules/proc_creating-a-rolling-content-view.adoc

```diff
@@ -0,0 +1,44 @@
+[id="creating-a-rolling-content-view"]
+= Creating a rolling content view
+
+Use this procedure to create a rolling content view.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-rolling-content-view[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Lifecycle* > *Content Views*.
+. Click *Create content view*.
+. In the *Create content view* window, enter a name for the content view in the *Name* field.
+{Project} automatically completes the *Label* field from the name you enter.
+. Optional: In the *Description* field, enter a description of the content view.
+. On the *Type* tab, select *Rolling content view*.
+. Click *Create content view*.
+. On the *Repositories* tab, select *All* from the *Status* list.
```

Comment:
Doesn't it also have a "Show repositories" button in the empty state? That would be an easier flow.

Reply:
Yes it does. -> reworded.